src/main/resources/application.yml

```diff
@@ -397,7 +480,7 @@ initializr:
           mappings:
           - versionRange: "[1.0.0.RELEASE,2.0.0.RELEASE)"
             version: 1.3.1.RELEASE
-          - versionRange: "[2.0.0.RELEASE,3.0.0.RELEASE)"
+          - versionRange: "[2.0.0.RELEASE,2.1.0.RELEASE)"
```

Comment:
No tasklauncher-cloudfoundry in 2.1.0.RELEASE.

Reply:
`- versionRange: "[2.0.0.RELEASE,2.1.0.RELEASE)"` means excluding `2.1.0.RELEASE`